CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+Thank you for contributing to this project!
+
+TL;DR
+Contributing Issues
+- Title:
+  - Be short and specific
+  - Add a word like 'Add' or 'Fix' to the beginning and capitalize it
+  - Do not add points, questionmarks to the title
+- Body:
+  - Do not tell your live story
+  - One picture, GIF or video tells more than a thousand words
+  - Be friendly
+  - Include nessary informations like versions, steps to reproduce, anything that could help
+- Comments:
+  - This is a discussion feature, use it
+  - BE FRIENDLY. You're the one how wants something, always keep that in mind
+  - If you email-reply watch out that you're not sending anything more than your text
+
+## Issues
+### Issue Title
+The issue title represents the issue, so please choose a name that fits. You cannot assign any tags 
+by yourself, but adding a word like 'Add' or 'Fix' as the first word really help to oversee the issues.
+Also, it's important to keep the name short enough to be readable as a whole without breaking the line.
+Not needed, but nice to have: Captitalize the first word. It looks better.
+### Issue Body
+Here you can be more specific as you were in the title, so please explain what has happened or what
+you want to add. Please write whole, grammaticaly correct, sentences and if you can, format your texts with 
+markdown. Also remember: One picture says more than a thousand words. In issues pics, gifs or videos can really help
+and are better then 'this thing there is broken. go fix it'. It also helps if you have a bit knowledge about what you're talking when you specify the problem. For example the taskbar on you PC is not called "The thing down there where you can start your apps". 
+### Comments
```

Comment:
Add section for reactions (:+1:, :-1:, etc), so if you read another issue and you agree with it, should add a reaction